### PR TITLE
Project overview tweaks to font sizes of names and headings.

### PR DESCRIPTION
**Pull request by sg00dwin**
First pass of ui adjustments
- Enable text truncation of long strings for resource names.
- Minor spacing changds for better alignment and prevent text blocks from touching.
- Font size & weight adjustments to match current origin-web-console overview.

* Follow on to align page heading and filter toolbar

tested in Chrome, Safari, FF, Edge

screen shot 2018-09-07 at 12 56 55 pm

screen shot 2018-09-07 at 12 55 11 pm

### CODE REVIEW

File: frontend/public/components/_project-overview.scss [hunk 1]
```diff
@@ -3,29 +3,38 @@
 }
 
 .project-overview__group-heading {
-  border-bottom: 1px solid black;
+  border-bottom: 1px solid $color-pf-black-400;
   margin: 0;
   padding: 5px 0;
+  font-size: 22px;
```
Comment (spadgett) [inline frontend/public/components/_project-overview.scss]: nit: alpha

File: frontend/public/components/project-overview.jsx [hunk 2]
```diff
@@ -13,17 +13,19 @@ const ProjectOverviewListItem = ({obj}) => {
...
+      className="co-resource-link-truncate"
       kind={kind}
       name={name}
       namespace={namespace}
-    />
+      />
   </h3>;
 
-  const additionalInfo = <div key={uid} className="project-overview__additional-info">
+const additionalInfo = <div key={uid} className="project-overview__additional-info">
```
Comment (spadgett) [inline frontend/public/components/project-overview.jsx]: indentation

File: frontend/public/components/_project-overview.scss [hunk 3]
```diff
@@ -3,29 +3,38 @@
...
   display: grid;
+  grid-column-gap: 10px;
   grid-template-columns: 50% 50%;
   grid-template-areas: "controller status";
   width: 100%;
 }
 
 .project-overview__detail--controller {
   grid-area: controller;
+  min-width: 0;
```
Comment (spadgett) [inline frontend/public/components/_project-overview.scss]: This could use a comment explaining why it's necessary.

**Comment by spadgett** [comment]
Can we a follow on, but we should move the filter to the right of the page heading

File: frontend/public/components/_project-overview.scss (hunk 3 above)
Comment (sg00dwin) [reply frontend/public/components/_project-overview.scss]: Needed to enable `text-overflow: ellipsis" within a css-grid area. I'll add comment.

File: frontend/public/components/_list-view.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.list-view-pf-description {
+  text-overflow: ellipsis;
+  overflow: hidden;
```
Comment (rhamilto) [inline frontend/public/components/_list-view.scss]: Nit: alpha

**Comment by spadgett** [comment]
```
/go/src/github.com/openshift/console/frontend/public/components/project-overview.jsx
  20:1  error  Expected indentation of 4 spaces but found 6  indent

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.
```

**Comment by sg00dwin** [comment]
@spadgett updated

**Pull request merged**